File: Tutorial_Sample_Code/DataConnectivity/DataSets/CS/CS_Win_Data_DataSets/DataSetConfiguration.cs
using System;
using System.Collections.Generic;
using System.Text;
using System.Data;
using System.Data.OleDb;

namespace CS_Win_Data_DataSets
{
    public class DataSetConfiguration
    {
        private const string CONNECTION_STRING = "Provider=Microsoft.Jet.OLEDB.4.0;Data Source=C:\\Xtreme.mdb";
        private const string QUERY_STRING = "SELECT * FROM CUSTOMER";
        private const string DATATABLE_NAME = "Customer";
        private const string DIRECTORY_FILE_PATH = "";

        public static DataSet CustomerDataSet
        {
            get
            {
                CustomerDataSetSchema dataSet = new CustomerDataSetSchema();
                OleDbConnection oleDbConnection = new OleDbConnection(CONNECTION_STRING);
                OleDbDataAdapter oleDbDataAdapter = new OleDbDataAdapter(QUERY_STRING, oleDbConnection);
                oleDbDataAdapter.Fill(dataSet, DATATABLE_NAME);
                return dataSet;
            }
        }
    }
}

File: Tutorial_Sample_Code/DataConnectivity/DataSets/VB/VB_Win_Data_DataSets/DataSetConfiguration.vb
Imports System.Data
Imports System.Data.OleDb

Public Class DataSetConfiguration

    Private Const CONNECTION_STRING As String = "Provider=Microsoft.Jet.OLEDB.4.0;Data Source=C:\xtreme.mdb"
    Private Const QUERY_STRING As String = "SELECT * FROM CUSTOMER"
    Private Const DATATABLE_NAME As String = "Customer"
    Private Const DIRECTORY_FILE_PATH As String = "C:\WebSites\VB_Web_Data_DataSets\"

    Public Shared ReadOnly Property CustomerDataSet() As DataSet
        Get
            Dim myDataSet As CustomerDataSetSchema = New CustomerDataSetSchema()
            Dim myOleDbConnection As OleDbConnection = New OleDbConnection(CONNECTION_STRING)
            Dim myOleDbDataAdapter As OleDbDataAdapter = New OleDbDataAdapter(QUERY_STRING, myOleDbConnection)
            myOleDbDataAdapter.Fill(myDataSet, DATATABLE_NAME)
            Return myDataSet
        End Get
    End Property



End Class
